content/content-tool/tool/src/webapp/vm/content/sakai_resources_list.vm

```diff
@@ -8,6 +8,9 @@
     vertical-align: middle !important;
     text-align:left !important;
   }
+  .table td{
+    text-indent: 0em !important;
+  }
```

Comment:
i don't understand this change. where is a text-indent being applied?

Reply:
In the "List of Dropboxes" table within the student view, there was an inline CSS code for text-indent: 1 em. I had to override that by adding !important. I have attached two screenshots with and without the text-indent for your reference. 
<img width="942" alt="W:o text-indent" src="https://github.com/user-attachments/assets/e855eae9-941b-4af6-9065-450a224cc231">
<img width="939" alt="With text-indent" src="https://github.com/user-attachments/assets/08781a6c-d8d8-4041-b919-e0c5a7c6f202">